.github/workflows/call-plugin-workflow.yml

```diff
@@ -0,0 +1,36 @@
+name: Run plugin workflow
+
+on:
+    pull_request:
+    workflow_dispatch:
+    push:
+        branches:
+            - master
+
+jobs:
+    call-test-workflow:
```

Comment:
please have a look at the original gitlab pipeline first. this plugin does not even have unit tests..